bugbug/bug_snapshot.py

```diff
@@ -868,17 +869,22 @@ def parse_flag_change(change):
     return bug
 
 
-def get_inconsistencies(bugs):
-    inconsistencies = []
+def _try_rollback(bug):
+    try:
+        rollback(bug, do_assert=True)
+        return None
+    except Exception:
+        logger.exception("Failed to rollback bug %s", bug["id"])
+        return bug
 
-    for bug in bugs:
-        try:
-            rollback(bug, do_assert=True)
-        except Exception:
-            logger.exception("Failed to rollback bug %s", bug["id"])
-            inconsistencies.append(bug)
 
-    return inconsistencies
+def get_inconsistencies(bugs):
+    with Pool() as p:
+        return [
+            bug
+            for bug in p.imap(_try_rollback, bugs, chunksize=1024)
+            if bug is not None
```

Comment:
```suggestion
            for bug, is_successful in p.imap(_try_rollback, bugs, chunksize=1024)
            if is_successful
```

If it were in GO, I would use `ok` instead of `is_successful` ;)